userguide/tutorials/catalog-examples.adoc

```diff
@@ -0,0 +1,380 @@
+
+[options="header",cols="1,1"]
+|===
+|Action   |Result   
+//-------------
+|Create a new account. Specify BCD as `25`  |Account created with BCD set to `25`
+|Create a subscription for the `standard-monthly` plan on `2021-09-17`   |Invoice generated for `$24.95` for `2021-09-17` to `2021-10-17` (BCD is ignored, subscription is billed as per its creation date)
+|Trigger invoice dry run with target date `2021-10-17`   |Invoice generated for `$24.95` for `2021-10-17` to `2021-11-17` 
+|Create a subscription for the `standard-annual` plan with date as `2021-09-30`   |No invoice generated
+|Trigger invoice dry run with target date `2021-09-30`   |Invoice generated for `$275` for `2021-09-30` to `2022-09-30` 
+|Trigger invoice dry run with target date `2021-10-17`   |Invoice generated for `$24.95` for `2021-10-17` to `2021-11-17` corresponding to `standard-monthly` plan 
+|Trigger invoice dry run with target date `2022-09-30`   |Invoice generated for `$275` for `2022-09-30` to `2023-09-30` corresponding to `standard-annual` plan  
+|===
+
+As mentioned above, even if there is an addon plan associated with a subscription, the addon plan will be invoiced as per its own date and NOT be aligned with the subscription billing date.
+
+=== Bundle Billing Alignment
+
+When `BUNDLE` billing alignment is specified in the catalog, all subscriptions in a bundle are invoiced together. The bill day is derived from the first bill date of the first subscription. 
+
+This example uses the  https://github.com/killbill/killbill-docs/blob/1a945188c0ef53a3c7f797b4890514d35c09c46f/catalogs/bundle-billing-alignment.xml[Bundle Billing Alignment] catalog. It specifies `BUNDLE` billing alignment. 
+
+
+It has a `BASE` and an `ADD_ON` product as follows:
+
+[options="header",cols="1,1"]
+|===
+|Product   |Category   
+//-------------
+|Standard   |Base   
+|RemoteControl   |Add_on   
+|===
+
+It has two plans corresponding to both products as follows:
+
+[options="header",cols="1,1,1"]
+|===
+|Plan Name   |Product|Plan Description
+//----------------------
+|standard_monthly   |Standard   |Single `EVERGREEN` phase with a monthly recurring charge of `$24.95`   
+|remotecontrol-monthly   |RemoteControl   |Single `EVERGREEN` phase with a monthly recurring charge of `$17.95`    
+|===
+
+The following table lists the actions and their results:
+
+[options="header",cols="1,1"]
+|===
+|Action   |Result   
+//-------------
+|Create a new account. Specify BCD as `25`  |Account created with BCD set to `25`
+|Create a subscription for the `standard-monthly` plan on `2021-09-20`   |Invoice generated for `$24.95` for `2021-09-20` to `2021-10-20` (BCD is ignored, subscription is billed as per its creation date)
+|Trigger invoice dry run with target date `2021-10-20`   |Invoice generated for `$24.95` for `2021-10-20` to `2021-11-20`
+|Add the `remotecontrol-monthly` add-on to the existing subscription with date as `2021-09-30`   |No invoice generated
```

Comment:
```suggestion
|Add the `remotecontrol-monthly` add-on to the existing subscription with date of `2021-09-30`   |No invoice generated
```

Reply:
So here the system does not auto-generate the invoice because it's an add-on product?

Reply:
@FlowWriter, no, the system does not auto-generate the invoice because it's a future date. I think we need to make this clear in the document. I'll make changes.

Reply:
Did you clarify this somewhere?